srcpkgs/devilutionX/template

```diff
@@ -1,16 +1,21 @@
 # Template file for 'devilutionX'
 pkgname=devilutionX
-version=1.2.1
+version=1.4.1
 revision=1
 build_style=cmake
-configure_args="-DVERSION_NUM=$version -DBINARY_RELEASE=ON -DTTF_FONT_PATH=\"/usr/share/fonts/truetype/CharisSILB.ttf\""
-makedepends="SDL2-devel SDL2_ttf-devel SDL2_mixer-devel libsodium-devel"
+configure_args="-DVERSION_NUM=$version -DBINARY_RELEASE=ON -DBUILD_TESTING=OFF -DDISABLE_ZERO_TIER=ON"
+hostmakedepends="git gettext pkg-config smpq"
+makedepends="SDL2-devel SDL2_image-devel fmt-devel libpng-devel libsodium-devel bzip2-devel"
 short_desc="Diablo I engine for modern operating systems"
 maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
 license="Unlicense"
 homepage="https://github.com/diasurgical/devilutionX"
 distfiles="https://github.com/diasurgical/devilutionX/archive/${version}.tar.gz"
-checksum=002dcbd4d4a5bdf8db1a3ec01139e5bfbed46d6a1caa32b17c9f2df161ad3521
+checksum=54b9fd496eba5b82d7e64891ab4de808f539c60b3b94bfa49639e0d9580fb7b8
+
+pre_configure() {
+	cp ${FILESDIR}/FindSDL2.cmake ${XBPS_BUILDDIR}/devilutionX-${version}/CMake/finders
+}
```

Comment:
In addition to using the file fetched from upstream (which requires `skip_extraction` to avoid confusing `xbps-src`), these kinds of file manipulations belong in `post_extract` so that it is possible to inspect the complete version of the source tree after `xbps-src -I extract <pkg>`. Also, avoid using absolute paths like `$XBPS_BUILDDIR` and instead rely on the fact that all of these functions will run in `${wkrsrc}`.

```suggestion
skip_extraction="FindSDL2.cmake"

post_extract() {
	cp ${XBPS_SRCDISTDIR}/${pkgname}-${version}/FindSDL2.cmake CMake/finders
}
```

Reply:
I agree; the less vendoring of files the better.

Reply:
Done!